modules/backend/formwidgets/codeeditor/assets/js/codeeditor.js

```diff
@@ -285,7 +285,7 @@
     }
 
     CodeEditor.prototype.onResize = function() {
-        this.editor.resize()
+        this.editor.trigger('resize')
```

Comment:
@ayumihamsaki I guess method that's called resize of the editor, not the winodw resize (but need to double-check).

Reply:
@w20k I've wait for you to confirm or not?

Reply:
Yeah, will confirm and post a comment ;) Thanks!

Reply:
@ayumihamsaki I was correct, it uses the ace.js resize method:
Usage:
https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/codeeditor/assets/vendor/ace/ace.js#L11988
Implementation:
https://github.com/octobercms/october/blob/master/modules/backend/formwidgets/codeeditor/assets/vendor/ace/ace.js#L15723

Could you revert all the expressions back to the old version: `this.editor.resize()`

Reply:
revert whole file